content/en/docs/languages/net/instrumentation.md

```diff
@@ -7,6 +7,20 @@ description: Instrumentation for OpenTelemetry .NET
 
 {{% docs/languages/manual-intro %}}
 
+{{% alert title="Note" color="info" %}}
+
+On this page you will learn how you can add traces, metrics and logs to your
+code manually. You are not limited to using one kind of instrumentation:
+you can also use [automatic instrumentation](/docs/languages/net/automatic/)
+to get started and then enrich your code with manual instrumentation as needed.
```

Comment:
confirm with auto-instrumentation owners, that it is feasible to have auto-instrumentation and enrichment via manual at the same time.

Reply:
See https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/main/docs/manual-instrumentation.md